src/live_restore/live_restore_fs.c

```diff
@@ -919,6 +913,64 @@ __wti_live_restore_fs_fill_holes(WT_FILE_HANDLE *fh, WT_SESSION *wt_session)
     return (ret);
 }
 
+/*
+ * __wti_live_restore_cleanup_tombstones --
+ *     Remove all tombstone files from the database.
+ */
+int
+__wti_live_restore_cleanup_tombstones(WT_SESSION_IMPL *session)
```

Comment:
This could do with a test. I think an `assert(len(glob.glob('WT_DEST/*.deleted') == 0)` at the end of `test_live_restore02` will do it.

Reply:
590fec3f1